101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Media Services Account."
+      }
+    },
+    "storageName": {
```

Comment:
Just to make sure here, I did `uniqueString(resourceGroup().id)`, is it what you are expecting?